src/assets/scss/_variables.scss

```diff
@@ -41,8 +41,7 @@ $body-bg:                      $gray-100;
 $font-family-sans-roboto:       'Roboto', sans-serif !default;
 $font-family-montserrat:        'Montserrat', sans-serif !default;
 $font-family-poppins:           'Poppins', sans-serif !default;
-$font-family-font-awesome:      'FontAwesome' !default;
-$font-family-font-fontello:     'fontello' !default;
+$font-family-matetial-icon:     'Material Design Icons' !default;
```

Comment:
Why we are not using $font-family-icon so we can assign $font-family-icon: $font-family-matetial-icon